### ISSUE TITLE
Apps: Only trigger the app manually

### ISSUE THREAD

**Issue by mattgwagner** [continuousit/seq-releases#333]
Running a newly setup version v2.3.3, decided to check out the HipChat app. 

The wording on the instance of an application is slightly confusing:

```
Only trigger the app manually (checked by default)

Events can be sent manually to any app from the drop-down menu in the event details. Uncheck this box to send events as they arrive. Signals can be used to select which events are sent
```

To me, this means that nothing should be sent to HipChat unless I specifically instruct it to. However, after installing the app, every event that went into Seq was forwarded into HipChat! I quickly disabled the app and started digging.

Unchecking that gives me options about what events to send into the app, which makes sense, I just don't think "Only trigger the app manually" is working as I would understand it.

**Comment by mattgwagner** [comment]
(Aside: love the product so far, along with Serilog. Fantastic work, and I'm hoping to get my org to purchase licensing once we finish evaluations!)

**Comment by nblumhardt** [comment]
Thanks for the report! This is a nasty bug that snuck through with some recent changes to Apps in Seq 2.3. Working on a patch and a new build now.

**Comment by nblumhardt** [comment]
Fixed in 2.3.4.

The perils of last-minute tweaks **:-|** ... Thanks again for reporting this @mattgwagner, 2.3.3 only went out a couple of days ago so great to catch this as early as you did.

Cheers!

**Comment by mattgwagner** [comment]
I updated the server and it worked like a charm. Thanks for the quick fix!
